From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @jimdial, has been notified to review your proposed change.

jimdial commented:
Thanks @4c74356b41.  #sign-off

andretms committed:
v1 of updating content, fixing broken links, unifying headers for build

andretms committed:
Updating ms.date for articles

TrishaMc committed:
edit pass: log-analytics-network-performance-monitor-pricing-faq

TrishaMc committed:
edit pass: log-analytics-network-performance-monitor-pricing-faq

TrishaMc committed:
edit pass: log-analytics-network-performance-monitor-pricing-faq

andretms committed:
Fixing a few format issues

andretms committed:
Update to formatting

andretms committed:
Updates to wording

andretms committed:
Change words for H2

andretms committed:
Fixing some formatting issues

andretms committed:
Text optimizations

andretms committed:
More minimal changes

andretms committed:
Fixing the name of claims controller

andretms committed:
More minor changes

andretms committed:
Minor revisions

andretms committed:
Missing a space

andretms committed:
Minor changes on sentence

TrishaMc committed:
edit pass: saas-dbpertenant-dr-geo-restore

TrishaMc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into saas-dbpertenant-dr-geo-restore

TrishaMc committed:
removing file

TrishaMc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into saas-dbpertenant-dr-geo-restore

TrishaMc committed:
edit pass: saas-dbpertenant-dr-geo-restore

TrishaMc committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into saas-dbpertenant-dr-geo-restore

TrishaMc committed:
edit pass: saas-dbpertenant-dr-geo-restore

TrishaMc committed:
edit pass: saas-dbpertenant-dr-geo-restore

TrishaMc committed:
edit pass: saas-dbpertenant-dr-geo-restore

andretms committed:
Merge branch 'master' into andret-build2018

andretms committed:
Updates to include files to add metadata

andretms committed:
Adding metadata to include files

andretms committed:
Merge branch 'master' into andret-build2018

neilpeterson committed:
aci-monitor

neilpeterson committed:
adding images

neilpeterson committed:
author

TrishaMc committed:
Merge branch 'master' into saas-dbpertenant-dr-geo-restore

andretms committed:
Merge branch 'master' into andret-build2018

andretms committed:
Merge from master

ausfeldt committed:
add 1.22.0

ausfeldt committed:
add 1.10.0

TrishaMc committed:
edit pass: saas-dbpertenant-dr-geo-restore

TrishaMc committed:
syncing changes

neilpeterson committed:
updated images / cli output

TrishaMc committed:
edit pass: saas-dbpertenant-dr-geo-restore

neilpeterson committed:
updated commands

mimig1 committed:
Cosmos DB: Deprecating MongoDB .NET tutorial & routing to QS

mimig1 committed:
Corrected an old redirection

mimig1 committed:
Actually removing file

neilpeterson committed:
Merge remote-tracking branch 'upstream/master' into aci-azure-monitor

neilpeterson committed:
copy update

neilpeterson committed:
testing new layout

neilpeterson committed:
multi-container-cli

neilpeterson committed:
images

neilpeterson committed:
coy update

neilpeterson committed:
updated copy

neilpeterson committed:
copy  updates

neilpeterson committed:
adding platform limitation note

neilpeterson committed:
code black decoration

rolyon committed:
Started activity logs portal section

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rbacreports-pr7266

Commit:
NW-MVC-1

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nw-mvc

Commit:
Fixes.

ronortloff committed:
Merge pull request #1 from MicrosoftDocs/master

Apr23Sync

neilpeterson committed:
Marsh feedback

mhko committed:
[Azure Search] Update references to MSDN docs.

mhko committed:
[Azure Search] Fix an incorrectly updated link.

neilpeterson committed:
adding image

neilpeterson committed:
playing around with dimensions

neilpeterson committed:
updated image

neilpeterson committed:
final copy updates

neilpeterson committed:
TOC

Commit:
Updates.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nw-mvc

Commit:
ToC fix.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nw-mvc

Commit:
Edits.

alkohli committed:
added info on pinout diagram - Github issue #7569

neilpeterson committed:
update var name

roygara committed:
Moving write accelerator article from SAP to Managed Disks content.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into moveAccel

Commit:
Link fix.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nw-mvc

roygara committed:
Removing article from SAP ToC

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into moveAccel

roygara committed:
Removing SAP variant of how to enable write accelerator article.

Commit:
fixes.

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nw-mvc

andretms committed:
fixing issues with includes

v-deken committed:
Bing News Search TOC changes, round 1

jodebrui committed:
clarifications

-what happens with ReadScale property in unsupported editions
-how to verify you are connected to read-only replica

rolyon committed:
Added information about PowerShell and CLI, deleted old screenshots

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rbacreports-pr7266

kevinlam1 committed:
Update logic-apps-control-flow-loops.md

Logic apps now supports nested loops, removed note sayin that we didn't

rolyon committed:
Tweaks to article and TOC

rolyon committed:
Tweaks to powershell heading

TrishaMc committed:
Merge pull request #38773 from alkohli/SSpinout

Pinout diagram - Github issue #7569

PRMerger18 committed:
Merge pull request #38793 from kevinlam1/patch-17

Update logic-apps-control-flow-loops.md

TrishaMc committed:
Merge pull request #38787 from v-deken/TOC-NewsSearch

Bing News Search TOC changes, round 1

ktoliver committed:
Merge pull request #38476 from neilpeterson/aci-azure-monitor

aci-monitor

charwen committed:
Moved LV from Level 3 to CenturyLink

charwen committed:
Moved LV from Level 3 to CenturyLink

ShawnJackson committed:
Merge pull request #37399 from TrishaMc/log-analytics-network-performance-monitor-pricing-faq2

edit pass: log-analytics-network-performance-monitor-pricing-faq

PRMerger14 committed:
Merge pull request #38794 from ausfeldt/patch-21

add 1.22.0

PRMerger14 committed:
Merge pull request #38795 from ausfeldt/patch-22

add 1.10.0

ShawnJackson committed:
Update saas-dbpertenant-dr-geo-restore.md

PRMerger16 committed:
Merge pull request #38798 from charwen/patch-56

Moved LV from Level 3 to CenturyLink

PRMerger17 committed:
Merge pull request #38711 from mhko/master

[Azure Search] Update MSDN links

ShawnJackson committed:
Update saas-dbpertenant-dr-geo-restore.md

tfosmark committed:
Merge pull request #38800 from charwen/patch-57

Moved LV from Level 3 to CenturyLink

ShawnJackson committed:
Merge pull request #37699 from TrishaMc/saas-dbpertenant-dr-geo-restore

edit pass: saas-dbpertenant-dr-geo-restore

anosov1960 committed:
Update sql-database-high-availability.md

Clarified read scale-out flag's behavior on standard/GP databases.

anosov1960 committed:
Update sql-database-high-availability.md

anosov1960 committed:
Update sql-database-high-availability.md

SnehaGunda committed:
Adding FAQ

ktoliver committed:
Merge pull request #38555 from andretms/andret-build2018-2

Guided Setup Updates for 2018 (2)

PRMerger17 committed:
Merge pull request #38808 from SnehaGunda/master

Adding FAQ

ktoliver committed:
fix relative path formatting for image

ktoliver committed:
Merge pull request #38679 from jimdial/nw-mvc

Network Watcher - MVC-1

tfosmark committed:
Merge pull request #38777 from roygara/moveAccel

Moving write accelerator doc from SAP to managed disks

tjvishnu committed:
Updated Geo and FAQ

Commit:
added visual studio link

Commit:
added link to prereq

Commit:
added section to upgrade the configuration server

Commit:
fixed a *

rolyon committed:
Added Azure to title and updated headings

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-rbacreports-pr7266

rolyon committed:
Small fixes

Sneezry committed:
Remove auto popup landing page

Sneezry committed:
Update iot-hub-arduino-iot-devkit-az3166-mqtt-helloworld.md

Sneezry committed:
Update iot-hub-arduino-iot-devkit-az3166-mqtt-helloworld.md

nsoneji committed:
updated with latest version v2.2

PRMerger20 committed:
Merge pull request #38819 from Sneezry/patch-1

Remove auto popup landing page

Sneezry committed:
Remove auto popup page

Sneezry committed:
Update iot-hub-arduino-iot-devkit-az3166-door-monitor.md

Sneezry committed:
Remove auto popup page

Commit:
Merge pull request #38823 from Sneezry/patch-2

Remove auto popup page

PRMerger10 committed:
Merge pull request #38824 from Sneezry/patch-3

Remove auto popup page

nsoneji committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

PRMerger14 committed:
Merge pull request #38826 from Sneezry/patch-4

Remove auto popup page

PRMerger14 committed:
Merge pull request #38818 from ruturaj/master

documentation checkins

nsoneji committed:
Added VVols support for VMware

nsoneji committed:
Added prefix Peak to Churn and IOPS to give the clarity

PRMerger13 committed:
Merge pull request #38817 from tjvishnu/patch-4

Updated Geo and FAQ

Commit:
Merge pull request #38825 from nsoneji/master

updated with latest version v2.2

bsiva committed:
Powershell changes

mayanknayar committed:
Updaing images and ip address examples

Updaing images and ip address examples in A2A networking article

Commit:
Expanded quickstarts node

Commit:
Merge pull request #38830 from bsiva/master

Powershell changes

ronortloff committed:
Repl tables GA update 1

update for replicated tables GA

tynevi committed:
Merge pull request #38610 from mimig1/04202018mongotutorial

Cosmos DB: Deprecating MongoDB .NET tutorial & routing to QS

Commit:
Fixed .Net link

PRMerger13 committed:
Merge pull request #38839 from ronortloff/master

Replicated tables updates

rjagiewich committed:
Merge pull request #38833 from mayanknayar/master

Updating images and ip address examples

ArLucaID committed:
Update how-to-use-vm-token.md

GitHubber17 committed:
Merge pull request #38835 from jimdial/patch-3

Expanded quickstarts node

Commit:
Merge pull request #38807 from anosov1960/patch-7

Update sql-database-high-availability.md

CarlRabeler committed:
Update sql-database-high-availability.md

PRMerger10 committed:
Merge pull request #38842 from arluca/patch-3

Update how-to-use-vm-token.md

martincoetzer committed:
Update active-directory-aadconnect-topologies.md

Adding hash and PTA

martincoetzer committed:
Update active-directory-aadconnectsync-attributes-synchronized.md

added hash and pta

rjagiewich committed:
Merge pull request #38840 from jimdial/patch-4

Fixed .Net link

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

itechedit committed:
Merge pull request #38671 from rolyon/rolyon-rbacreports-pr7266

[Azure RBAC] Update access report article to use Activity Log issue 7266

JeffGoldner committed:
Updated list of supported OSes

martincoetzer committed:
Update active-directory-aadconnect-accounts-permissions.md

added hash

PRMerger15 committed:
Merge pull request #38844 from martincoetzer/patch-8

Update active-directory-aadconnect-topologies.md

PRMerger16 committed:
Merge pull request #38845 from martincoetzer/patch-9

Update active-directory-aadconnectsync-attributes-synchronized.md

Commit:
quickfix06

martincoetzer committed:
Update active-directory-passwords-overview.md

Added hash and PTA

v-alje committed:
Merge pull request #38848 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger19 committed:
Merge pull request #38843 from CarlRabeler/patch-54

Update sql-database-high-availability.md

martincoetzer committed:
Update active-directory-passwords-troubleshoot.md

added pass-through authentication

PRMerger20 committed:
Merge pull request #38855 from martincoetzer/patch-11

Update active-directory-passwords-overview.md

GitHubber17 committed:
Merge pull request #38851 from JeffGoldner/master

Updated list of supported OSes

PRMerger20 committed:
Merge pull request #38850 from martincoetzer/patch-10

Update active-directory-aadconnect-accounts-permissions.md

Commit:
Merge pull request #38854 from MarkusVi/quickfix06

quickfix06

martincoetzer committed:
Update active-directory-aadconnectsyncservice-features.md

added hash

Commit:
Merge pull request #38857 from martincoetzer/patch-12

Update active-directory-passwords-troubleshoot.md

Commit:
Merge pull request #38788 from jodebrui/patch-14

clarifications

jodebrui committed:
clarification about behavior of connections if readscale is not enabled/supported

PRMerger10 committed:
Merge pull request #38859 from martincoetzer/patch-13

Update active-directory-aadconnectsyncservice-features.md

PRMerger14 committed:
Merge pull request #38866 from jodebrui/patch-16

clarification about behavior of connections if readscale is not enabl…

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPrivateRepo

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
